pom.xml

```diff
@@ -350,6 +351,12 @@
         <version>${jooby.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.jstach</groupId>
```

Comment:
io.jooby

Reply:
I probably should have opened intellij or eclipse instead of using vim :)